Dockerfile

```
@@ -1,14 +1,14 @@
-## ====================== DOWNLOAD DEPENDENCIES ===============================
-
-#FROM sonar
-FROM sonarqube:7.9.1-community
+FROM sonarqube:7.9.3-community AS base
```

If there is only one FROM directive, I don't see the need to use a multistage build syntax.
We should remove the "AS base" part from this line.
```suggestion
FROM sonarqube:7.9.3-community
```